pages/linux/kpartx.md

```diff
@@ -0,0 +1,15 @@
+# kpartx
+
+> Create device maps from partition tables.
+
+- Add partition mappings:
+
+`kpartx -a {{whole_disk.img}}`
+
+- Delete partition mappings:
+
+`kpartx -d {{whole_disk.img}}`
+
+- List partition mappings that would be added -a:
```

Comment:
What does `-a` mean in this description?

Also, no need to force-push - we've got a button that does it for us when merging :smiley_cat:

Reply:
I've copied the explanation from the man page :see_no_evil: I simplified the explanation :wink: